streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -498,6 +498,9 @@ void commit(final boolean startNewTransaction) {
             }
         } catch (final CommitFailedException | ProducerFencedException error) {
             throw new TaskMigratedException(this, error);
+        } catch (final TimeoutException error) {
+            log.error("Main thread times out while doing commit.");
```

Comment:
Can you elaborate on what this means? It doesn't appear to contain more information than the error itself. What are we trying to convey here?